pallets/ethereum-events/src/lib.rs

```diff
@@ -1707,4 +1686,32 @@ pub mod migrations {
         log::info!("ℹ️  Migrated Ethereum event's NFT contract addresses successfully");
         return consumed_weight
     }
+
+    pub fn migrate_to_bridge_contract<T: Config>() -> frame_support::weights::Weight {
+        sp_runtime::runtime_logger::RuntimeLogger::init();
+        log::info!("ℹ️  Ethereum events pallet data migration invoked");
+
+        let mut consumed_weight = Weight::from_ref_time(0);
+
+        let address = LiftingContractAddress::<T>::get();
+        let validators_address = ValidatorManagerContractAddress::<T>::get();
+        let pb_address = PublishRootContract::<T>::get();
+
+        log::info!("ℹ️  Checking if zero");
+        if !address.is_zero() && !validators_address.is_zero() && !pb_address.is_zero() {
```

Comment:
```suggestion
        if !lift_address.is_zero() && !validators_address.is_zero() && !pub_root_address.is_zero() {
```

Reply:
Let's try to be consistent with variable names. We had 3 contract addresses, and each address followed a different convention:
- just `address` (could have been `address1, address2, address3`)
- properly qualified address (with `validators` indicating usage)
- old style short variable name, with letters _suggesting_ usage but not quite (`pb_address`). I know it's local variables, but no need to require a different mindset in every line.

Reply:
The other thing about this block is that it introduces a strong dependency between the 3 addresses. The data migration will only happen if all 3 addresses are defined.
I don't mind the code being like this if we check the state of the chain before migrating, since this is a one-off, but if any of the chains (from test to mainnet) turns out to have conditions that will stop this migration, then this has to be changed.